src/schema/nmdc.yaml

```diff
@@ -256,6 +256,28 @@ classes:
       - has_solution_components
       - volume
 
+  ChemicalReagent:
```

Comment:
we already have modeling for this. I'll post it in a minute

Reply:
Resolution: make ChemicalReagent specific to the fact that it is used in a laboratory. like "LaboratoryChemicalReagent" or "LaboratoryReagent"